### PR TITLE
WK Generic 4 step ph cycle

### PR DESCRIPTION
**Pull request by wfkincaid**

### CODE REVIEW

File: examples/run_largecyc_generic_echo.py
Comment (jmfranck) [inline examples/run_largecyc_generic_echo.py:65]: See comments on slack on the new way to name these.. show the phase argument above, and name them according to the conjugate coherence domain, not according to how the phase is actually cycled

File: examples/run_largecyc_generic_echo.py
Comment (jmfranck) [inline examples/run_largecyc_generic_echo.py:136]: I have not compared against the original generic echo sequence... Be sure to do this if you have not

File: examples/run_largecyc_generic_echo.py
Comment (jmfranck) [reply examples/run_largecyc_generic_echo.py:65]: The new CPMG sequence in my pull request is an example of how to do this

File: examples/run_largecyc_generic_echo.py
Comment (jmfranck) [inline examples/run_largecyc_generic_echo.py:7]: This description of the phase cycling no longer applies... Give the phase argument here as well like I did in my pull request.

File: SpinCore_pp/simple_fns.py
Comment (jmfranck) [inline SpinCore_pp/simple_fns.py:23]: see that this makes sense to you -- edit if not

File: examples/run_largecyc_CPMG.py
Comment (jmfranck) [inline examples/run_largecyc_CPMG.py:36]: mostly, this is stylistic

but for clarity, we should get `r_` from numpy -- it's in pyspecdata, but just comes from numpy

File: examples/active.ini
Comment (jmfranck) [inline examples/active.ini:3]: I changed the base branch so it merges to master.  In general, before merging the PR, we should just checkout active.ini from master, unless there's an exceptional reason to change something

File: examples/run_largecyc_CPMG.py
Comment (jmfranck) [inline examples/run_largecyc_CPMG.py:10]: I reviewed all this -- this was nicely done

File: examples/run_largecyc_generic_echo.py
Comment (jmfranck) [inline examples/run_largecyc_generic_echo.py:10]: I reviewed all this -- this was nicely done

File: examples/run_largecyc_CPMG.py
Comment (jmfranck) [inline examples/run_largecyc_CPMG.py:2]: Unfortunately merging my branch into this branch was a bad move.  We need to finalize my pull request first.

Comment (guinneak198) [reply examples/run_largecyc_CPMG.py:2]: the idea I had was merging this into your branch then merging to master but we won't do that anymore

Comment (jmfranck) [reply examples/run_largecyc_CPMG.py:2]: but remember -- we need to keep PRs so that they only do *one thing at a time* and are *as simple as possible*. 

Now, we need to wait for the other PR to get merged before we can merge this, or else comments, etc, for the other guy get lost.

File: SpinCore_pp/ppg/generic.py
Comment (jmfranck) [inline SpinCore_pp/ppg/generic.py:34]: you need to merge master into this branch to get rid of extra  diffs  now

File: examples/run_largecyc_generic_echo.py
Comment (jmfranck) [inline examples/run_largecyc_generic_echo.py:22]: do not import array like this -- use `np.array`

File: SpinCore_pp/ppg/generic.py
Comment (jmfranck) [reply SpinCore_pp/ppg/generic.py:34]: (on a PR, if you update master, the website won't update until you merge master into the branch)

**Review by guinneak198 — APPROVED** [review]
looks good

**Pull request merged**
